railties/lib/rails/generators/rails/plugin/templates/Rakefile

```diff
@@ -13,17 +13,13 @@ RDoc::Task.new(:rdoc) do |rdoc|
   rdoc.rdoc_files.include('README.md')
   rdoc.rdoc_files.include('lib/**/*.rb')
 end
-
```

Comment:
I prefer to be explicit about the blank lines. Let's add the `-%>` in our ERB tags and keep the blank lines in the template.